techniques/general/G212.html

```diff
@@ -2,133 +2,110 @@
-            <li><a href="#description">Description</a></li>
-            <li><a href="#examples">Examples</a></li>
-            <li><a href="#resources">Related Resources</a></li>
-            <li><a href="#tests">Tests</a></li>
+            <li><a href="#important-information">達成方法に関する重要な情報</a></li>
+            <li><a href="#applicability">適用 (対象)</a></li>
+            <li><a href="#description">解説</a></li>
+            <li><a href="#examples">事例</a></li>
+            <li><a href="#resources">参考リソース</a></li>
+            <li><a href="#tests">検証</a></li>
          </ul>
       </nav>
-      <h1>Using native controls to ensure  functionality is triggered on the up-event.</h1>
+      <h1>ネイティブコントロールを使用して、アップイベントで機能がトリガーされるようにする</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://w3c.github.io/wcag/understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://w3c.github.io/wcag/understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>This technique applies to nearly all technologies.</p>
-            <p>This technique relates to <span><a href="https://w3c.github.io/wcag/understanding/pointer-cancellation">Success Criterion 2.5.2: Pointer Cancellation</a> (Sufficient)</span>.
+            <h2>適用 (対象)</h2>
+            <p>この達成方法は、ほぼ全ての技術に適用される。</p>
+            <p>これは<span><a href="https://w3c.github.io/wcag/understanding/pointer-cancellation">達成基準 2.5.2: ポインタのキャンセル</a> (十分な達成方法) </span>に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to ensure that users who attempt to interact with
-               a control do not trigger the action of the event accidentally. This can be accomplished
-               most directly by relying on the pointer's up-event (for example, the onclick or mouseup
-               event).  
+            <h2>解説</h2>
+            <p>この達成方法の目的は、あるコントロールとインタラクションしようとする利用者が、イベントのアクションを誤ってトリガーしないようにすることである。これは、ポインタのアップイベント (例えば、onclick 又は mouseup イベント) に頼ることで最も直接的に達成できる。
```

Comment:
actionは動作をしていたと思いますので。
```suggestion
            <p>この達成方法の目的は、あるコントロールとインタラクションしようとする利用者が、イベントの動作を誤ってトリガーしないようにすることである。これは、ポインタのアップイベント (例えば、onclick 又は mouseup イベント) に頼ることで最も直接的に達成できる。
```